lncfg/protocol_legacy_on.go

```diff
@@ -20,6 +20,10 @@ type ProtocolOptions struct {
 	// Anchors should be set if we want to support opening or accepting
 	// channels having the anchor commitment type.
 	Anchors bool `long:"anchors" description:"EXPERIMENTAL: enable experimental support for anchor commitments, won't work with watchtowers"`
+
+	// WumboChans should be set if we want to disable support for wumbo
+	// channels, which is the opposite of mini.
+	WumboChans bool `long:"wumbo" description:"if set, then lnd will create and accept requests for wumbo channels"`
```

Comment:
i'm not sure this is the right place for this config option, perhaps it should be on its own subconfig? this is currently only available in `dev` builds since originally this subconfig was intended for testing backwards compatibility with legacy protocol options. it seems like with anchors that delineation got mangled a bit (which can be seen from the filename).

imo things like anchors and wumbo should really be separate from legacy things that we only retain for testing purposes. i don't see a reason to expose `LegacyOnionFormat` or `CommitmentTweak` in prod

naming suggestion: `wumbo` is imo not self-descriptive enough, `wumbo-channels` is a little better but still somewhat ambiguous. i wonder also if one should be able to configure acceptance separately from creation. could do something like `accept-wumbo-channels` and `create-wumbo-channels`

Reply:
>  this is currently only available in dev builds since originally this subconfig was intended for testing backwards compatibility with legacy protocol options.

another way to achieve this is to have a `Legacy` struct nested within the `ProtocolOptions`, that itself can be empty in prod. also gives a nice naming scheme `--protocol.legacy. committweak`